geppetto-ui/src/menu/MenuSection.js

```diff
@@ -140,7 +140,24 @@ class MenuSection extends React.Component {
           aria-describedby={id}
           onClick={this.handleClick}
           onMouseOver={this.handleOver}
-          onMouseOut={this.handleOut}
+          onMouseOut={this.handleOut}>
+          { this.props.button.icon !== ""
+            ? this.props.button.caret
+              ? this.props.button.caret.show
+                ? <span style={ { color : this.props.button.activeColor } }>
+                  <i className={this.props.button.icon}/>
+                  <i 
+                    className={this.props.button.caret.icon ? this.props.button.caret.icon : "fa fa-caret-down"}
```

Comment:
I think it's better to set icons by passing the icon component as a prop rather than a class name. There are Geppetto applications requiring alternative icons that cannot be included by specifying the class only